src/depthcloud/DepthCloud.js

```diff
@@ -29,11 +29,10 @@ ROS3D.DepthCloud = function(options) {
   this.pointSize = options.pointSize || 3;
   this.width = options.width || 1024;
   this.height = options.height || 1024;
+  this.resolutionFactor = Math.max(this.width, this.height) / 1024;
```

Comment:
Is there something special about 1024 besides being the default height/width?  Should this be `max / min` instead of `max / 1024`?

Reply:
> Should this be max / min instead of max / 1024?

No. It should be a ratio of the default 1024 resolution. The `max` is just for completeness, we can use either the width or height, since the encoded image is always square.